test/scala/com/github/johnynek/bazel_deps/ParseTest.scala

```diff
@@ -29,6 +29,7 @@ class ParseTest extends FunSuite {
                 Some(Set("core", "args", "date").map(Subproject(_))),
                 None,
                 None,
```

Comment:
we should add some explicit tests for classifiers.

Reply:
Added some simple tests for parsing/handling classifiers

The normalize implementation backing the failing ParseGeneratedDocTest is a monster, and not something that I can see clearly through to understanding properly. Perhaps you can tell me why my change would logically cause it to fail the `p.modules == None` requirement out of hand.